source/docs/deployment.md

```diff
@@ -70,6 +70,74 @@ Option | Description
 `repo`, `repository` | Heroku repository URL
 `message` | Customize commit message (Default to `Site updated: {% raw %}{{ now('YYYY-MM-DD HH:mm:ss') }}{% endraw %}`)
 
+## Netlify
+
+[Netlify](https://www.netlify.com/) provides continuous deployment services, global CDN, ultra-fast DNS, atomic deploys, instant cache invalidation, automatic HTTPS, a browser-based interface, a CLI, and many other features for managing your Hexo website. 
+
+Go to https://app.netlify.com/signup and select your preferred signup method. This will likely be a hosted Git provider, although you also have the option to sign up with an email address.
+
+There are two different ways to deploy your sites on Netlify.
+
+1. Use the web UI
```

Comment:
An idea is to merge the `###` headings with the step number. e.g. `### 1. Create a Netlify account`

Reply:
Or eschew numbers is also good